packages/syft/src/syft/core/adp/data_subject_list.py

```diff
@@ -633,12 +633,7 @@ def rint(self, *args: Any, **kwargs: Any) -> DataSubjectArray:
     def __array_ufunc__(self, ufunc, method, *inputs, **kwargs) -> ArrayLike:  # type: ignore
```

Comment:
Great Find @kiendang , I just experimented and by the NumPy docs (https://numpy.org/doc/stable/reference/arrays.classes.html)
MAYBE : we could remove `__array_unfunc__` in the data subject array, as we do not explicitly override the unfuncs?

Reply:
Created #7125 for that.